### PR TITLE
GH-4091 changes based on release review

### PR DESCRIPTION
**Pull request by hmottestad**
GitHub issue resolved: # 

Briefly describe the changes proposed in this PR:

----
PR Author Checklist (see the [contributor guidelines](https://github.com/eclipse/rdf4j/blob/main/.github/CONTRIBUTING.md) for more details):

 - [ ] my pull request is [self-contained](https://rdf4j.org/documentation/developer/merge-strategy/#self-contained-changes-pull-requests-and-commits)
 - [ ] I've added tests for the changes I made
 - [ ] I've applied [code formatting](https://github.com/eclipse/rdf4j/blob/main/.github/CONTRIBUTING.md#code-formatting) (you can use `mvn process-resources` to format from the command line)
 - [ ] I've [squashed](https://rdf4j.org/documentation/developer/squashing) my commits where necessary 
 - [ ] every commit message starts with the issue number (GH-xxxx) followed by a meaningful description of the change

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -3,22 +3,42 @@
...
+fulfills the DCO's requirement that you sign-off on your contributions.
+
+For more information, please see the Eclipse Committer Handbook:
+https://www.eclipse.org/projects/handbook/#resources-commit
+
+### Signing the Eclipse Contributor Agreement
+You must digitally sign the [Eclipse Contributor Agreement (ECA)](https://www.eclipse.org/legal/ECA.php). You can do this as follows:
+
+* If you haven't done so already, [register an Eclipse account](https://accounts.eclipse.org/user/register). **Important**: Use the same email address that you will use on Git commits as the author address.
+* Open the [ECA form](https://accounts.eclipse.org/user/eca) and complete it. See the [ECA FAQ](https://www.eclipse.org/legal/ecafaq.php) for more info.
```
Comment (hmottestad) [inline CONTRIBUTING.md:40]: Updated and moved the CONTRIBUTING.md file.

File: compliance/elasticsearch/src/test/java/org/eclipse/rdf4j/sail/elasticsearch/ElasticsearchSailGeoSPARQLTest.java [hunk 2]
```diff
@@ -4,6 +4,8 @@
  * are made available under the terms of the Eclipse Distribution License v1.0
  * which accompanies this distribution, and is available at
  * http://www.eclipse.org/org/documents/edl-v10.php.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
```
Comment (hmottestad) [inline compliance/elasticsearch/src/test/java/org/eclipse/rdf4j/sail/elasticsearch/ElasticsearchSailGeoSPARQLTest.java:8]: Added SPDX line.

**Comment by hmottestad** [comment]
I've also added the SPDX lines to the old Aduna copyright statements. Is that correct @jeenbroekstra ?

**Comment by jeenbroekstra** [comment]
> I've also added the SPDX lines to the old Aduna copyright statements. Is that correct @jeenbroekstra ?

Yep that should be fine - the license is the same, it's just the copyright notice that's different.

**Review by jeenbroekstra — APPROVED** [review]

**Pull request merged**